### DISCUSSION TITLE
Error during `Palmers Indices` Computation

### DISCUSSION THREAD

**Discussion by utkarsh-1912** [monocongo/climate_indices#523] [Q&A]
I am facing the ` 'PicklingError("Can't pickle <ufunc '_cafec_coeff_ufunc'>: attribute lookup _cafec_coeff_ufunc on __main__ failed")' `
![image](https://github.com/monocongo/climate_indices/assets/79002186/8a3d4c2d-f770-4647-8b1c-ba8c71d4d7a9)
I am computing indices for one location (1 lat and 1 lon ) value pair. Is error occurring due to it ? 

Please tell the solution for it.

**Comment by monocongo** [comment]
My apologies, @utkarsh-1912 the Palmer indices provided by this package are experimental only and as such are no longer supported. Please see the note on the project README. If you find open source implementation of PDSI etc. in Python, please let us know so we can direct users to that package. I'm not aware of such an alternative, but the code herein may provide a reasonable launch point for such an effort.
